src/app/api/proxy/route.ts

```diff
@@ -0,0 +1,33 @@
+import { ticket } from "@/contracts/events/ticket"
+import { type NextRequest, NextResponse } from "next/server"
+import { z } from "zod"
+import axios from "axios"
+
+const EventDto = z.object({
+  eventId: z.string(),
+  aggregator: z.string(),
+  eventType: z.string(),
+  validTime: z.string(),
+  payload: z.any(),
+})
+
+export async function POST(request: NextRequest) {
+  try {
+    const json: unknown = await request.json()
+    const body = EventDto.parse(json)
+    if (body.aggregator === ticket.flowType) {
+      await axios.post("http://localhost:3000/api/transform/ticket", body, {
```

Comment:
Is this proxy route specific to the ticket? or can it be used for anything?
Would be great with a little comment block above the method explaining the purpose of this method :D